src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -539,45 +541,49 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
               sx={{ borderBottom: `1px solid ${theme.palette.gray[600]}` }}
             >
               <Tooltip title="Previous Run" arrow>
-                <Button
-                  sx={{ fontSize: '28px', px: 0, flex: 1 }}
-                  variant="text"
-                  onClick={() => {
-                    // This might be confusing, but index 0 is the most recent run, so incrementing the index goes
-                    // to an *earlier* run.
-                    dispatch(selectResultIdx(selectedResultIdx + 1));
-                    navigate(
-                      `?workflowDagResultId=${
-                        workflow.dagResults[selectedResultIdx + 1].id
-                      }`
-                    );
-                  }}
-                  disabled={
-                    selectedResultIdx === workflow.dagResults.length - 1
-                  }
-                >
-                  <FontAwesomeIcon icon={faChevronLeft} />
-                </Button>
+                <Box sx={{ px: 0, flex: 1 }}>
```

Comment:
To resolve this error (tooltip won't display without this):
![image](https://user-images.githubusercontent.com/30596854/223541430-41b23c36-7b30-4cb4-b46f-d3f4bf82b5e4.png)